includes/class-notification.php

```diff
@@ -661,6 +661,8 @@ public function replace_all_fields( $text = '' ) {
                 } else {
                     $table .= '&mdash;';
                 }
+            }elseif (in_array( $value['type'], [ 'google_map' ] ) ) {

```

Comment:
Spacing. Change:
`}elseif (in_array( $value['type'], [ 'google_map' ] ) ) {`
to:
`} elseif ( in_array( $value['type'], ['google_map'] ) ) {`

Please see [space usage](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage).

----

[Use long array syntax](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#declaring-arrays). Change `[ 'google_map' ]` to `array( 'google_map' )`

Reply:
![image](https://user-images.githubusercontent.com/5925563/118804651-f5874e80-b872-11eb-8ddc-479fba826296.png)